### DISCUSSION TITLE
Problem on my synology

### DISCUSSION THREAD

**Discussion by vincentBesseau** [Koenkk/zigbee2mqtt#19031] [General]
From two days, I updated dsm 7.1 to 7.2 but after this update, my sonoff-P is on error : 

`error 2023-09-19 07:14:34: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)`

I tryed to update the driver with https://github.com/robertklep/dsm7-usb-serial-drivers

And it partialy work, my zwave has been restored but I alway the same issue for zigbee "SRSP - SYS ...".

I update my sonoff has usual, but again this issue.
I buying another sonoff-P without update and again and again :-(

Someone can help me, i'm loosing ?

**Comment by lux73** [comment]
i'm sorry but this is unlikely a z2m Problem

if it worked before the Synology Update it seems the Update itself is the Problem. I suggest to ask the Synology Community Forum.

**Reply by vincentBesseau** [reply]
After some analysis, the problem was with Gladys assistant.
It seems that this assistant is not completely compatible with Z2M. They are working to resolve this issue.
Sorry for this post.

[github.com/GladysAssistant](https://github.com/GladysAssistant/Gladys/issues/1887)

[community.gladysassistant.com](https://community.gladysassistant.com/t/zigbee2mqtt-et-synology-ds920-en-7-2-probleme-suite-upgrade-synology/8365/13?u=spenceur)
